addons/binding/org.openhab.binding.allplay/src/main/java/org/openhab/binding/allplay/internal/AllPlayBindingProperties.java

```diff
@@ -17,25 +17,34 @@
  */
 public class AllPlayBindingProperties {
 
-    private final Integer rewindSkipTimeInSec;
-    private final Integer fastForwardSkipTimeInSec;
+    private final double rewindSkipTimeInSec;
```

Comment:
Indeed I didn't have the latest version of ESH. Unfortunately, upgrading didn't help. applyParameterType is only called when I open the config dialog in the PaperUI, not when I save the new values. The other two adjusted functions in eclipse/smarthome#4001 are only calling parseInt on the default value, not on the user-entered value. Seems like the fix is only for default values?